pkg/cmd/pipelinerun/describe_test.go

```diff
@@ -271,8 +264,52 @@ Taskruns
 NAME   TASK NAME   STARTED         DURATION    STATUS
 tr-1   t-1         8 minutes ago   3 minutes   Succeeded
 `
-	if d := cmp.Diff(expected, actual); d != "" {
-		t.Errorf("Unexpected output mismatch: %s", d)
+	tu.AssertOutput(t, expected, actual)
+}
+
+func TestPipelineRunDescribe_without_start_time(t *testing.T) {
+	clock := clockwork.NewFakeClock()
+
+	cs, _ := pipelinetest.SeedTestData(t, pipelinetest.Data{
+		PipelineRuns: []*v1alpha1.PipelineRun{
+			tb.PipelineRun("pipeline-run", "ns",
+				cb.PipelineRunCreationTimestamp(clock.Now()),
+				tb.PipelineRunLabel("tekton.dev/pipeline", "pipeline"),
+				tb.PipelineRunSpec("pipeline"),
+				tb.PipelineRunStatus(
+					tb.PipelineRunStatusCondition(apis.Condition{
+						Status: corev1.ConditionUnknown,
+						Reason: resources.ReasonRunning,
+					}),
+				),
+			),
+		},
+	})
+
+	p := &test.Params{Tekton: cs.Pipeline, Clock: clock}
+
+	pipelinerun := Command(p)
+	clock.Advance(10 * time.Minute)
+	actual, err := test.ExecuteCommand(pipelinerun, "desc", "pipeline-run", "-n", "ns")
+	if err != nil {
+		t.Errorf("Unexpected error: %v", err)
 	}
+	expected := `Name:           pipeline-run
+Namespace:      ns
+Pipeline Ref:   pipeline
 
+Status
+STARTED   DURATION   STATUS
+---       ---        Running
```

Comment:
@hrishin shouldn't be the status sthg else than `Running` if it didn't start ? :stuck_out_tongue_closed_eyes:

Reply:
yeah :smile_cat: , will fix that (just got lazy)